src/prefect/main.py

```diff
@@ -1,4 +1,7 @@
 # Import user-facing API
+from typing import Any
+
+from pluggy import Result
```

Comment:
I think this came in by accident and is causing the failures in CI.

Reply:
Yup, just found it too. Not sure how my IDE managed this.